File: Hopa-master/Hopa-master/app/src/main/java/com/poly/hopa/models/Service.java
package com.poly.hopa.models;

import com.denzcoskun.imageslider.models.SlideModel;
import com.google.gson.annotations.SerializedName;

import java.lang.reflect.Array;
import java.util.ArrayList;
import java.util.List;

public class Service {
    @SerializedName("_id")
    private String serviceID;
    private int price;
    private String name;
    private String description;
    private List<String> image;

    public String getServiceID() {
        return serviceID;
    }

    public void setServiceID(String serviceID) {
        this.serviceID = serviceID;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public List<String> getImage() {
        return image;
    }

    public void setImage(List<String> image) {
        this.image = image;
    }

    public boolean isFavorite() {
        return false;
    }
}

File: ServerHotel-WebManager/ServerHotel-WebManager/models/Service.js
const mongoose = require('mongoose');

const schema = mongoose.Schema;

const serviceSchema = new schema({


    name: {
        type: String,
        require: true
    },
    price: {
        type: Number,
        require: false
    },
    description: {
        type: String,
        require: false
    },
    image: {
        type: Array,
        required: true
    }
});

const Service = mongoose.model('Service', serviceSchema);

module.exports = Service;